db.js

```diff
@@ -1,14 +1,17 @@
 var nano = require('nano'),
 	url = require('url'),
-    path = require('path');
+    path = require('path'),
+    request = require('request');
```

Comment:
How has this ever worked without request before?

Reply:
As below.